prometheus/generic/setup-haproxy-exporter/tasks/docker.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Run the haproxy_exporter
+  docker_container:
+    name: haproxy_exporter
+    image: "{{ haproxy_exporter_image }}:{{ haproxy_exporter_image_version }}"
+    pid_mode: "host"
+    network_mode: "host"
+    state: "{{ provision_state }}"
+    command: "--haproxy.scrape-uri=\"http://{{ stats_user }}:{{ stats_password }}@127.0.0.1:{{ stats_port }}/stats;csv\""
```

Comment:
Do you see a need to also support `https`?

Reply:
It's definitely good point to consider, but I wouldn't say it's necessary, because the stats page is accessed only locally. This covers the scenario when the exporter is running on the haproxy node. Also the prometheus.yml.j2 was written with this in mind (the scraping target points to the loadbalancers ['ansible_ssh_host'] ansible variable). Https would make sense in case we want to monitor remote loadbalancers, but this would also require  further changes in the prometheus.yml config.